adoc/bp_chap_troubleshooting.adoc

```diff
@@ -312,6 +312,72 @@ For more information see also:
+|===
+|Report | Invoked as | Description
+| Channel Packages | [command]``channel-packages`` | List of packages in a channel.
+| Channel Report | [command]``channels`` | Detailed report of a given channel.
+| Cloned Channel Report | [command]``cloned-channels`` | Detailed report of cloned channels.
+| Custom Info | [command]``custom-info`` | System custom information.
+| Entitlements | [command]``entitlements`` | Lists all organizations on SUSE Manager with their system or channel entitlements.
+| Patches in Channels | [command]``errata-channels`` | Lists of patches in channels.
+| Patches Details | [command]``errata-list`` | Lists all patches that affect systems registered to SUSE Manager.
+| All patches | [command]``errata-list-all`` | Complete list of all patches.
+| Patches for Systems | [command]``errata-systems`` | Lists applicable patches and any registered systems that are affected.
+| Host Guests | [command]``host-guests`` | List of host-guests mapping.
+| Inactive Systems | [command]``inactive-systems`` | List of inactive systems.
+| System Inventory | [command]``inventory`` | List of systems registered to the server, together with hardware and software information.
+| Kickstart Trees | [command]``kickstartable-trees`` | List of kickstartable trees.
+| All Upgradable Versions | [command]``packages-updates-all`` | List of all newer package versions that can be upgraded.
+| Newest Upgradable Version | [command]``packages-updates-newest`` | List of only newest package versions that can be upgraded.
+| Result of SCAP | [command]``scap-scan`` | Result of OpenSCAP sccdf eval.
+| Result of SCAP | [command]``scap-scan-results`` | Result of OpenSCAP sccdf eval, in a different format.
+| System Data | [command]``splice-export`` | System data needed for splice integration.
+| System Groups | [command]``system-groups`` | List of system groups.
+| Activation Keys for System Groups | [command]``system-groups-keys`` | List of activation keys for system groups.
+| Systems in System Groups | [command]``system-groups-systems`` | List of systems in system groups.
+| System Groups Users | [command]``system-groups-users`` | Report of system groups users.
+| Installed Packages | [command]``system-packages-installed`` | List of packages installed on systems.
+| Users in the System | [command]``users`` | Lists all users registered to SUSE Manager.
+| Systems administered | [command]``users-systems`` | List of systems that individual users can administer.
+|===
+
+For more information about an individual report, run [command]``spacewalk-report`` with the option [option]``--info`` or [option]``--list-fields-info`` and the report name.
+The description and list of possible fields in the report will be shown.
+
+For further information, the [literal]``spacewalk-report(8)`` man page as well as the [option]``--help``parameter of the [command]``spacewalk-report`` program can be used to get additional information about the program invocations and their options.
```

Comment:
"For further information on program invocations and options, see ... "

Reply:
thanks! so much better